packages/nns/src/governance.canister.spec.ts

```
@@ -524,6 +530,10 @@ describe("GovernanceCanister", () => {
         include_empty_neurons_readable_by_caller: [],
         // The field is present in the argument but ignored by the old service.
         include_public_neurons_in_full_neurons: [],
+        // The field is present in the argument but ignored by the old service.
```

This is temporary, right?
We don't want to silently ignore parameters.
We should use the new service if these parameters are present, like we do with `includeEmptyNeurons`.

---

This refers to the test case for the old service. We will implement pagination in a new PR.